test/integration_test/snapshot_test.go

```diff
@@ -94,9 +94,13 @@ func groupSnapshotTest(t *testing.T) {
 	require.Len(t, allVolumeNames, 3, "Should only have two volumes and 1 snapshot")
 
 	dataVolumesNames := make([]string, 0)
+	dataVolumesInUse := make([]string, 0)
 	for _, volume := range allVolumeNames {
 		if !strings.HasPrefix(volume, "snapshot") {
 			dataVolumesNames = append(dataVolumesNames, volume)
+			if volume == "mysql-data-1" {
```

Comment:
The volume name will be pvc-<UUID>, right?

Reply:
Oh actually, the above check is incorrect. It again passed on my cluster since the data replicas get placed differently. Let me fix this.

Reply:
Fixed